.github/ISSUE_TEMPLATE/2024-08-06-introducing-layer5-learning-paths.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Introducing Layer5 Learning Paths"
```

Comment:
@Shubhf you still adding a blog under .github folder . why? This is a workflow directory not a blog